src/core/util/sys_vars.cpp

```diff
@@ -1149,6 +1153,10 @@ void mce_sys_var::get_env_params()
         zc_cache_threshold = option_size::from_str(env_ptr);
     }
 
+    if ((env_ptr = getenv(SYS_VAR_TX_QUEUE_EXPANSIONS))) {
```

Comment:
make the env similar to our internal sys var, it is weird that one contains burst for example, and the other one does not...